in_text/text_02_drivers.yml

```diff
@@ -351,47 +367,46 @@ entities:
       attr-def: >-
         Discharge, in cubic meters per second
       attr-defs: This data release
-      data-min: 0.002831685
-      data-max: 784.3767
+      data-min: 0.003
+      data-max: 784.377
```

Comment:
I updated min/max based on what I saw in the data, but I noted some concerns about data accuracy for temp in the conversation thread. This should be reviewed before values are updated.